.pre-commit-config.yaml

```diff
@@ -24,10 +24,11 @@ repos:
       - id: prettier
         files: \.(json|yaml|yml)$
 
-  - repo: https://github.com/koalaman/shellcheck-precommit
-    rev: v0.9.0
+  - repo: https://github.com/codespell-project/codespell
```

Comment:
it appears codespell is in here twice now and shellcheck has been removed, I think you pasted it over shellcheck instead of over the existing codespell config - let me know if I am mistaken

Reply:
https://github.com/facebookincubator/ForgeArmory/blob/b8d4594c4a50da6c07da96328ec19b888d584a3a/.pre-commit-config.yaml

Reply:
You're right, doh!